tools/ubuntu-setup/all.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
+
+#  This script can be tested for validity by doing something like:
+#
+#  docker run -v "${OPENWHISK_HOME}:/openwhisk" ubuntu:trusty \
+#    sh -c 'apt-get update && apt-get -y install sudo && /openwhisk/tools/ubuntu-setup/all.sh'
```

Comment:
So good you added this comment

Reply:
Yeah.  It took me a little time to line everything up, but it makes for many fewer breaks in production.